lib/groups_page.rb

```diff
@@ -4,9 +4,21 @@
 class GroupsPage < SobraniePage
   decorator Scraped::Response::Decorator::AbsoluteUrls
 
+  CEASED_MEMBERS_URL = 'mps-whose-mandate-has-not-been-completed-2014-2018'
```

Comment:
Hard-coding the dates here means that this page will only work for this single term, even though this should be a class that works for all historic terms too…